README.md

```diff
@@ -420,10 +420,41 @@ On linux, you will need to increase the available [virtual memory](https://www.e
 
 To delete the containers and **all the data** stored in the deployed CDCS system, run:
 
+```shell
+docker-compose down -v
```

Comment:
I would also add the simpler `docker-compose down` and specify that data can be kept that way!